documentation/how-to/tracking.en.md

```diff
@@ -69,3 +69,23 @@ on a legend layer linked to an active tracking session and select the
 *Stop tracking* button.
 
 !![](../assets/images/tracking-stop.png)
+
+## Predefined Project Tracking Sessions
+
+This functionality enables one or more position tracking sessions to automatically start upon project load. Users will be presented with a feature form popup as the tracking session begins to fill in attributes. If the vector layer used to track has the 'Hide Form on Add Feature' mode selected, the feature form will be skipped. 
+
+!![Hide Form on Added Feature](../assets/images/hide-form-on-add-feature.png)
+
+### Configure a Project Tracking Session
+:material-monitor: Desktop preparation
+
+The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the qfieldsync plugin is installed in your QGIS profile.
```

Comment:
On line 75 we use single quotes, here we use double quotes. Is there any kind of convention or something?

Reply:
```suggestion
The configuration of a project tracking session happens in the vector layer properties dialog's QField panel. There, you can activate "Tracking Session" and specify the tracking requirements. Note that the QField panel is only available when the QFieldSync plugin is installed in your QGIS profile.
```

Reply:
I think is better with double quote!